apps/linkedin-to-notion/src/background/index.ts

```diff
@@ -4,16 +4,17 @@ export {};
 
 // You can test the regex here: https://regex101.com/r/RJgYar/1
 // Don't forget to update the comment when you update the regex on regex101.com
-export const linkedInURLRegex = /linkedin\.com\/in\/[^/]+\/#?$/;
+export const linkedInURLRegex = /linkedin\.com\/in\/[^/]+\/#?/;
```

Comment:
This is needed because when LinkedIn changes url, it starts with a weird url like `https://www.linkedin.com/in/nije-üô/?mini_profile...` so it closes the panel that we want to keep open when switching from one profile to another